src/Prestige.Identity.WebApi/Controllers/CommonController.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace Prestige.Identity.WebApi.Controllers
+{
+    public class CommonController : ControllerBase
```

Comment:
Maybe rename the `CommonController` to `BaseController`? Sounds more generic and all of your controllers will inherit that.